### PR TITLE
Enable remote config tests for PHP

### PR DESCRIPTION
**Pull request by estringana**
## Description

Enable remote config tests for PHP

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [ ] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [ ] CI is passing
- [ ] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

File: tests/remote_config/test_remote_configuration.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def rc_check_request(data, expected, caching):
 
 @rfc("https://docs.google.com/document/d/1u_G7TOr8wJX0dOM_zUDKuRJgxoJU_hVTd5SeaMucQUs/edit#heading=h.octuyiil30ph")
 @released(cpp="?", dotnet="2.15.0", golang="1.44.1", java="0.115.0")
-@released(php="?", python="1.7.0rc1.dev", ruby="?", nodejs="3.9.0")
+@released(php="0.81.0", python="1.7.0rc1.dev", ruby="?", nodejs="3.9.0")
```
Comment (Anilm3) [inline tests/remote_config/test_remote_configuration.py:132]:
```suggestion
@released(php_appsec="0.7.0", python="1.7.0rc1.dev", ruby="?", nodejs="3.9.0")
```

File: utils/build/docker/php/common/php.ini [hunk 2]
```diff
@@ -25,3 +25,8 @@ ddappsec.helper_socket_path=/tmp/ddappsec.sock
 ddappsec.helper_lock_path=/tmp/ddappsec.lock
 ddappsec.helper_extra_args=--log_level debug
 ddappsec.helper_log_file=/tmp/helper.log
+
+datadog.appsec.enabled_on_cli=0
+datadog.trace.agent_port=8126
```
Comment (estringana) [inline utils/build/docker/php/common/php.ini:30]: On php, default agent port is `0` and on system tests it is expected to be `8126`

**Review by robertomonteromiguel — APPROVED** [review]

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
code format issue (lint check failed)

**Comment by cbeauchesne** [comment]
Ci is failing on PHP, I mark this PR as draft, you can put it back as normal PR when issue are solved.

File: utils/proxy/rc_mocked_responses_asm_activate_only.json [hunk 3]
```diff
@@ -10,6 +10,6 @@
         "raw": "eyJhc20iOnsiZW5hYmxlZCI6dHJ1ZX19"
       }
     ],
-    "targets": "eyJzaWduZWQiOnsiZXhwaXJlcyI6IjIwMjItMDktMTdUMTI6NDk6MTVaIiwic3BlY192ZXJzaW9uIjoiMS4wLjAiLCJjdXN0b20iOnsib3BhcXVlX2JhY2tlbmRfc3RhdGUiOiJhYWFhYSJ9LCJ0YXJnZXRzIjp7ImRhdGFkb2cvMi9BU01fRkVBVFVSRVMvYXNtX2ZlYXR1cmVzX2FjdGl2YXRpb24vY29uZmlnIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjE1OTY1OGFiODViZTcyMDc3NjFhNDExMTE3MmIwMTU1ODM5NGJmYzc0YTFmZTFkMzE0ZjIwMjNmN2M2NTZkYiJ9LCJsZW5ndGgiOjI0fX0sInZlcnNpb24iOjIzMzM3MzkzfSwic2lnbmF0dXJlcyI6W3sia2V5aWQiOiJURVNUX0tFWV9JRCIsInNpZyI6ImRhODhmZTg3NzVjNjM4MzFiNGU3Nzk2MWIyYTk5NzhkYjM2YTNmZGUzMWExZmJjMGMyMTA2ZWRmMWQ4ZDJkYjI2NzAyN2U4NTI0YzI2NTg5MDhkMzA1Yjc3YTQ5NmE5ZjFkMWM1ZWQ2MjFiNmMwZjA2MTA4OTk5Y2FlOGFmOTBlIn1dfQo="
```
Comment (estringana) [inline utils/proxy/rc_mocked_responses_asm_activate_only.json:13]: The encoded targets here is missing the field `signed` - `_type` which according to the documentation is mandatory and its value has to be targets. This new encoded strings is the same as the previous one but with the that field added

Comment (smola) [reply utils/proxy/rc_mocked_responses_asm_activate_only.json:13]: Just for the record. Before:

```json
{
  "signed": {
    "expires": "2022-09-17T12:49:15Z",
    "spec_version": "1.0.0",
    "custom": {
      "opaque_backend_state": "aaaaa"
    },
    "targets": {
      "datadog/2/ASM_FEATURES/asm_features_activation/config": {
        "custom": {
          "v": 1
        },
        "hashes": {
          "sha256": "159658ab85be7207761a4111172b01558394bfc74a1fe1d314f2023f7c656db"
        },
        "length": 24
      }
    },
    "version": 23337393
  },
  "signatures": [
    {
      "keyid": "TEST_KEY_ID",
      "sig": "da88fe8775c63831b4e77961b2a9978db36a3fde31a1fbc0c2106edf1d8d2db267027e8524c2658908d305b77a496a9f1d1c5ed621b6c0f06108999cae8af90e"
    }
  ]
}
```

After:
```json
{
  "signed": {
    "_type": "targets",
    "expires": "2022-09-17T12:49:15Z",
    "spec_version": "1.0.0",
    "custom": {
      "opaque_backend_state": "aaaaa"
    },
    "targets": {
      "datadog/2/ASM_FEATURES/asm_features_activation/config": {
        "custom": {
          "v": 1
        },
        "hashes": {
          "sha256": "159658ab85be7207761a4111172b01558394bfc74a1fe1d314f2023f7c656db"
        },
        "length": 24
      }
    },
    "version": 23337393
  },
  "signatures": [
    {
      "keyid": "TEST_KEY_ID",
      "sig": "da88fe8775c63831b4e77961b2a9978db36a3fde31a1fbc0c2106edf1d8d2db267027e8524c2658908d305b77a496a9f1d1c5ed621b6c0f06108999cae8af90e"
    }
  ]
}
```

**Comment by estringana** [comment]
@cbeauchesne it passed all tests. Back to ready for review

**Review by cbeauchesne — APPROVED** [review]

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**